src/routers/v2/DepartNotification.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import type Request from 'express';
+import ApplicationRouter from '../../appdev/ApplicationRouter';
+import NotificationUtils from '../../utils/NotificationUtils';
+
+class DepartNotification extends ApplicationRouter<Array<Object>> {
+  constructor() {
+    super(['POST']);
+  }
+
+  getPath(): string {
+    return '/departNotification/';
+  }
+
+  // eslint-disable-next-line require-await
+  async content(req: Request): Promise<any> {
+    if (
+      !req.body
```

Comment:
Don't need to change it in this PR but we should look into using a library for validating so we avoid huge if statements next semester @alanna-zhou . I would look into https://www.npmjs.com/package/validate or https://validatejs.org/